Core/include/Components/AudioComponent.h

```diff
@@ -1,34 +1,34 @@
-﻿#ifndef AUDIOCOMPONENT_H
-#define AUDIOCOMPONENT_H
+﻿#ifndef AUDIOCOMP_H
```

Comment:
Optional Nitpick: Why not just use pragma once

Reply:
personal preference.